busy_beaver/blueprints/slack/slash_command.py

```diff
@@ -1,16 +1,21 @@
 import logging
 from typing import List, NamedTuple
-from urllib.parse import urlencode
+
 import uuid
+from urllib.parse import urlencode
 
 from flask import request
 from flask.views import MethodView
 
 from .toolbox import make_slack_response
+
 from busy_beaver.apps.upcoming_events.workflow import (
     generate_next_event_message,
     generate_upcoming_events_message,
 )
+
+from busy_beaver.apps.github_summary.workflow import (generate_account_attachment, delete_account_attachment)
+
```

Comment:
please delete the line breaks

Reply:
Resolved in this [commit](https://github.com/busy-beaver-dev/busy-beaver/pull/173/commits/039291fd1ddde082b825d3c5fda4b1a5d1b17e16)